src/main/java/com/yurjinia/common/exception/CommonException.java

```diff
@@ -14,16 +16,30 @@ public class CommonException extends RuntimeException {
     private HttpStatus status;
 
     public CommonException(ErrorCode errorCode, HttpStatus status) {
-        setErrorCode(errorCode);
-        setStatus(status);
-        this.params = List.of(errorCode.getDescription());
+        this.errorCode = errorCode;
+        this.status = status;
+        this.params = List.of();
     }
 
     public CommonException(ErrorCode errorCode, HttpStatus status, List<String> params) {
-        this.params = new ArrayList<>();
-        setErrorCode(errorCode);
-        setParams(params);
-        setStatus(status);
+        this.errorCode = errorCode;
+        this.status = status;
+        this.params = params;
     }
 
+    @Override
+    public String getMessage() {
+        return getMessageForLocale(Locale.ENGLISH);
+    }
+
+    @Override
+    public String getLocalizedMessage() {
+        return getMessageForLocale(LocaleContextHolder.getLocale());
+    }
+
+    private String getMessageForLocale(Locale locale) {
+        return LocalizedMessagesUtil
+                .getMessageForLocale(errorCode.name(), locale, params.toArray(new String[]{}))
```

Comment:
`params.toArray(new String[]{}))` - do we need to use it like this?
every request it's going to create an array. is there a better approach of using params as array?